src/classes/CourseConnectionSettingsModel.cls

```diff
@@ -0,0 +1,47 @@
+/*
+    Copyright (c) 2021, Salesforce.org
+    All rights reserved.
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+    * Redistributions of source code must retain the above copyright
+    notice, this List of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+    notice, this List of conditions and the following disclaimer in the
+    documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+    its contributors may be used to endorse or promote products derived
+    from this software without specific prior written permission.
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2021
+* @group 
+* @group-content ../../ApexDocContent/
+* @description Model class for Course Connection Recordtypes
+*/
+public with sharing class CourseConnectionSettingsModel {
+
+    // Properties for this model
+    public Boolean courseConnectionsEnabled;
+    public String studentRecordTypeId;
+    public String facultyRecordTypeId;
+
+    // Constructor to set the values
+    public CourseConnectionSettingsModel(Boolean courseConnectionsEnabled, String studentRecordTypeId, String facultyRecordTypeId) {
+        this.courseConnectionsEnabled = courseConnectionsEnabled;
+        this.studentRecordTypeId = studentRecordTypeId;
+        this.facultyRecordTypeId = facultyRecordTypeId;
```

Comment:
This wasn't in the spec but is a newly-revealed issue. Please apply a ternary on String.isBlank(value) to set it to '""' if that is true, otherwise use the passed value.